conf/flowswitch.config

```diff
@@ -13,42 +13,45 @@ params {
     run_trim_galore_fastqc = true
```

Comment:
Why having this file separate from the base `nextflow.config`?
It seems to me that all logic regarding params should be in the same file for easier readability.
But I understand why it was put there for keeping files simple too.
@nf-core/core any comments on that?

Reply:
Yes I just felt like the flow logic should be in a different file. It makes it easier to read for me but its not a big deal with change this